docs/content/docs/tasks_usage/observability/tracing.md

```diff
@@ -35,21 +35,56 @@ The sections below outline how to make required updates depending on whether you
 
 If you already have OSM running, `tracing` values must be updated in the OSM ConfigMap using:
 
-```console
-# Replace osm-system with osm-controller's namespace if using a non default namespace
-kubectl patch ConfigMap osm-config -n osm-system -p '{"data":{"tracing_enable":"true", "tracing_address":"<tracing server hostname>", "tracing_port":"<tracing server port>", "tracing_endpoint":" <tracing server endpoint>"}}' --type=merge
+> Note: Replace `osm-system` in the commands below with osm-controller's namespace if using a non default namespace
+
+1. Enable tracing.
+```bash
+kubectl patch ConfigMap osm-config \
+  --namespace osm-system \
+  --type=merge \
+  --patch '{"data":{"tracing_enable": "true"}}'
 ```
+
+2. Provide the **address** of the Jaeger instance to the OSM Controller. (Example: `jaeger.osm-system.svc.cluster.local`)
+```bash
+kubectl patch ConfigMap osm-config \
+  --namespace osm-system \
+  --type=merge \
+  --patch '{"data":{"tracing_address": "<tracing server hostname>"}}'
+```
+
+3. Provide the **port** of the Jaeger instance to the OSM Controller. (Example: `9411`)
+```bash
+kubectl patch ConfigMap osm-config \
+  --namespace osm-system \
+  --type=merge \
+  --patch '{"data":{"tracing_port": "<tracing server port>"}}'
+```
+
+4. Provide the **endpoint** of the Jaeger instance to the OSM Controller. (Example: `/api/v2/spans`)
+```bash
+kubectl patch ConfigMap osm-config \
+  --namespace osm-system \
+  --type=merge \
+  --patch '{"data":{"tracing_endpoint": "<tracing server endpoint>"}}'
+```
+
+
 > Note: To make this change persistent between upgrades, see osm mesh upgrade --help.
 
 You can verify these changes have been deployed by inspecting `osm-config`:
-```console
```

Comment:
how come you changed these from console to bash?

Reply:
Typically for snippets that only contain commands and no output (as in this case), I use the `bash` identifier (`shell` also works). This allows to copy-paste as is without needing to exclude any shell command delimiters such as `$`. 

I only use `console` if the snippet contains the command output as well, so it is well-formatted and is rendered as a console would, with coloring.

So for this particular usage, `bash` seems more appropriate than `console` to me. I could be wrong in the way I approach this though, so open to hearing your thoughts.

Reply:
That's a good question...
(time passes....)
I made this this demo to show it to myself whether this changes anything. I think it does.
It seems that `console` and `bash` render the code block differently.
Here is my example (https://gist.github.com/draychev/79b62878628aca9bf22ddcc3bca93854):

![image](https://user-images.githubusercontent.com/49918230/112687781-3ba4d080-8e35-11eb-9cce-8997cab79fdb.png)

And in particular for this doc:
Original: 
![image](https://user-images.githubusercontent.com/49918230/112688212-c5549e00-8e35-11eb-997f-bfa703ed5e31.png)

New: 
![image](https://user-images.githubusercontent.com/49918230/112688290-e0271280-8e35-11eb-9b94-0ba1310bf4df.png)
(the json blob is subtly different then than the rest)


Checking the supported list of grammars: https://github.com/github/linguist/blob/master/vendor/README.md
Actually here: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5353

`console` seems to be the alias for `ShellSession` - https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5353
and `bash` is for `Shell` - https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5282-L5284

I actually do not fully understand the difference and won't dig deeper.

@michelleN I'm happy to revert back to `console` if you don't think the visual change is compelling?

Reply:
Thanks for the explanation. I actually do the same thing as @shashankram explained and these visuals you provided are really helpful. I'm good with keeping them as bash. Seems more appealing to me.